Source/EasyNetQ.Logging.Microsoft/LoggerFactory.cs

```diff
@@ -0,0 +1,52 @@
+using MS = Microsoft.Extensions.Logging;
+using MSExtensions = Microsoft.Extensions.Logging.LoggerExtensions;
+using MSFactoryExtensions = Microsoft.Extensions.Logging.LoggerFactoryExtensions;
+
+namespace EasyNetQ.Logging.Microsoft
+{
+    /// <summary>
+    ///     Creates ILogger
+    /// </summary>
+    public interface ILoggerFactory
```

Comment:
I don't fully understand, why is this interface duplicated in other projects too?

Reply:
Agree. It is enough to have `ILogger/<T>` interfaces and their implementations that depend on appropriate logger classes. No need in factories/~Wrappers.

Reply:
Fixed